sdk/file_sdk/Common/file_format.h

```diff
@@ -13,26 +13,19 @@ class FileFormat : public IFileFormat
 public:
   virtual string GetOriginalExtension();

```

Comment:
> virtual string GetOriginalExtension(); [](start = 1, length = 39)

string GetOriginalExtension() override;

don't use virtual keyword and use override in overridden methods